src/oled_display_node.cpp

```diff
@@ -864,32 +871,35 @@ int main(int argc, char **argv)
     hostname = getPopen("uname -n");
     dispError |= dispOled_writeText(&g_oledDisplayCtx, DISP_LINE_HOSTNAME, 0, DISP_TEXT_START_MODE, hostname.c_str());
     ros::Duration(updateDelay).sleep();
-    getMainIpAddress(displayedIpAddress, 0);
-    dispError |= dispOled_writeText(&g_oledDisplayCtx, DISP_LINE_IP_ADDR, 0, DISP_TEXT_START_MODE, displayedIpAddress.c_str());
+
+    getIpAddressses(displayedETHAddress, displayedWLANAddress, 0);
+    dispError |= dispOled_writeText(&g_oledDisplayCtx, DISP_LINE_IP_WLAN, 0, DISP_TEXT_START_MODE, displayedWLANAddress.c_str());
+    ros::Duration(updateDelay).sleep();
+    dispError |= dispOled_writeText(&g_oledDisplayCtx, DISP_LINE_IP_ETH, 0, DISP_TEXT_START_MODE, displayedETHAddress.c_str());
     ros::Duration(updateDelay).sleep();
 
     // If there is a battery_state topic and we get the callback also show battery voltage
     if (g_batteryVoltage > 0.0) {
-        ROS_DEBUG("%s Battery voltage is now %4.1f volts.", THIS_NODE_NAME, g_batteryVoltage);
+      ROS_DEBUG("%s Battery voltage is now %4.1f volts.", THIS_NODE_NAME, g_batteryVoltage);
 
-        // Control what shows up after the voltage reading.
-        // If you change the text, use same number of chars so display does not move around on the line
-        std::stringstream stream;
-        stream << std::fixed << std::setprecision(1) << g_batteryVoltage;
-	if (g_batteryVoltage >= BAT_CHARGING_LEVEL) {
-            stream <<   " CHRG";
-        }else if (g_batteryVoltage >= BAT_LOW_LEVEL) {
-            stream <<   " OK  ";
-        } else {
-          if ((loopIdx & 1) == 0) {
-              stream << " LOW ";
-          } else {
-              stream << "     ";
-          }
-        }
-        std::string battText = "Bat " + stream.str();
-        dispError |= dispOled_writeText(&g_oledDisplayCtx, DISP_LINE_BATT_VOLTS, 1, DISP_TEXT_START_MODE, battText.c_str());
-        ros::Duration(updateDelay).sleep();
+      // Control what shows up after the voltage reading.
+      // If you change the text, use same number of chars so display does not move around on the line
+      std::stringstream stream;
+      stream << std::fixed << std::setprecision(1) << g_batteryVoltage;
+
+      if (g_batteryVoltage <= BAT_LOW_LEVEL && (loopIdx % 2) == 0){
+        stream << "V LOW";
+      }
+      else if (g_batteryVoltage >= BAT_CHARGING_LEVEL && (loopIdx % 3) == 0){
```

Comment:
PROBLEM HERE  I just ran on charger for yet another case.   Don't show the percentage when in CHRG display.  It is of near zero value and as it stands right now you get this packed display that is all run together.  It ends up when 100% is showing having   27.8V100%  so looks very bad.
The purpose of 'blinking' when LOW was to attract attention.   When voltage is in 'Charging' range just show CHRG and don't show percent.  My experience with these batteries of assorted ages is the voltage it sits at while charging is of little value it can read 29.2V for super weak battery but not at all be done charging.     If you really feel percentage is important to you when at high V charging then you will need to fix the 'V' that leads to all letters run together for 100% display.   
Also the  V should be gotten rid of anyway.  Our display is so cramped for space  saying V is not useful because the V is implied and our doc on this should say that value is volts.